### PR TITLE
Scorecard components and reviewers

### PR DESCRIPTION
**Pull request by cseed**
Add components and list of reviewers for those components to the index.html page.  It looks like this:

screen shot 2019-02-14 at 11 07 43 am

The order of the names is random each time the page is loaded.  Idea is to take the first name for the component you're reviewing when creating PRs.

@danking I removed you from Hail front-end.  Now everyone appears on 2-3 components.

@akotlar FYI, I ripped out the JSON endpoints since I'm happy with the Flask implementation.

### CODE REVIEW

**Comment by akotlar** [comment]
@cseed Ok that's too bad.

File: scorecard/scorecard/scorecard.py [hunk 1]
```diff
@@ -65,34 +59,20 @@
...
-    random_user = random.choice(users)
+    def functional_shuffle(x):
+        x = list(x)
+        random.shuffle(x)
+        return x
 
+    component_users_random = {c: functional_shuffle(us) for c, us in component_users.items()}
     return render_template('index.html', unassigned=unassigned,
-                           user_data=user_data, urgent_issues=urgent_issues, random_user=random_user, updated=updated)
+                           user_data=user_data, urgent_issues=urgent_issues, component_users=component_users_random, updated=updated)
```
Comment (danking) [inline scorecard/scorecard/scorecard.py]: copy_shuffle is perhaps more descriptive
```suggestion
    def copy_shuffle(x):
         x = list(x)
         random.shuffle(x)
         return x

 
      component_users_random = {c: copy_shuffle(us) for c, us in component_users.items()}
     return render_template('index.html', unassigned=unassigned,		     return render_template('index.html', unassigned=unassigned,
                           user_data=user_data, urgent_issues=urgent_issues, component_users=component_users_random, updated=updated)
```

Comment (danking) [reply scorecard/scorecard/scorecard.py]: hmm suggestions don't work for multiple lines.

File: scorecard/scorecard/scorecard.py [hunk 2]
```diff
@@ -65,34 +59,20 @@
 def index():
     user_data, unassigned, urgent_issues, updated = get_users()
 
-    random_user = random.choice(users)
+    def functional_shuffle(x):
+        x = list(x)
+        random.shuffle(x)
+        return x
 
+    component_users_random = {c: functional_shuffle(us) for c, us in component_users.items()}
```
Comment (jigold) [inline scorecard/scorecard/scorecard.py]: Rather than returning the full list, why not shuffle and then return the first element?

Comment (cseed) [reply scorecard/scorecard/scorecard.py]: No reason.  I can change it if you prefer.

Comment (cseed) [reply scorecard/scorecard/scorecard.py]: Changed it to include one user.  Also obviates @danking's comment.

**Pull request merged**